src/client/containers/Home/Home.js

```diff
@@ -1,3 +1,9 @@
 import React from 'react';
+import Button from '../../components/ReactToastify/Button/Button';
 
-export const Home = () => <section className="home">Final project!</section>;
+export const Home = () => (
+  <section className="home">
+    Final project!
+    <Button />
```

Comment:
Just add the toast part here, no need to introduce a component just for testing stuff since it will be deleted anyway.